Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -61,6 +61,17 @@ def get_test_playbook_ids(self):
 
         return test_ids
 
+    def get_marketplacev2_ids(self):
+        marketplacev2_tests = self._conf['test_marketplacev2']
+        test_ids = []
+        pack_ids = []
+
+        for t in marketplacev2_tests:
+            test_ids.append(t['playbookID'])
+            pack_ids.append(t['packID'])
```

Comment:
We usually don't mention pack names in the conf.json. Do we have to have it hardcoded? why not take it from the id set?